router/http.go

```diff
@@ -238,7 +238,23 @@ func (h *httpSyncHandler) Set(data *router.Route) error {
 	service.refs++
 	r.service = service
 	h.l.routes[data.ID] = r
-	h.l.domains[strings.ToLower(r.Domain)] = r
+	if data.Path == "/" {
+		log.Printf("router: adding default route for domain: %s", r.Domain)
+		if tree, ok := h.l.domains[strings.ToLower(r.Domain)]; ok {
+			log.Printf("router: default route exists, overwriting")
+			tree.backend = r
+		} else {
+			log.Printf("router: doesn't yet exist, inserting")
```

Comment:
I'm thinking of removing those entirely now that I look at them, they were mostly useful when debugging.